pandas/tests/io/excel/test_readers.py

```diff
@@ -1332,6 +1332,16 @@ def test_euro_decimal_format(self, read_ext):
         )
         tm.assert_frame_equal(result, expected)
 
+    def test_read_boolean_column_with_blanks(self, read_ext):
+        # GH 45903
+        result = pd.read_excel(
+            "boolean_with_blanks" + read_ext, dtype={"var1": "boolean"}
+        )
+        expected = DataFrame(
+            [[True], [False], [None], [True]], columns=["var1"], dtype="boolean"
```

Comment:
```suggestion
            {'var1': [True, False, None, True}, dtype="boolean"
```
This seems simpler and easier to read and maintain.